src/zerocash/Address.h

```diff
@@ -44,19 +45,19 @@ class PublicAddress {
 public:
     /* This constructor is to be used ONLY for deserialization. */
     PublicAddress();
-    PublicAddress(const std::vector<unsigned char>& a_pk, std::string& pk_enc);
+    PublicAddress(const uint256& a_pk, uint256& pk_enc);
     PublicAddress(const PrivateAddress& addr_sk);
 
     bool operator==(const PublicAddress& rhs) const;
     bool operator!=(const PublicAddress& rhs) const;
 
 
-    const std::vector<unsigned char>& getPublicAddressSecret() const;
-    const std::string getEncryptionPublicKey() const;
+    const uint256& getPublicAddressSecret() const;
```

Comment:
Note that this is just something we should clean up, not necessarily in this set of changes.